src/prefect/agent.py

```diff
@@ -442,7 +443,13 @@ async def submit_run(self, flow_run: FlowRun) -> None:
                 self.logger.exception(
                     f"Failed to get infrastructure for flow run '{flow_run.id}'."
                 )
-                await self._propose_failed_state(flow_run, exc)
+                await self._propose_crashed_state(
+                    flow_run,
+                    (
+                        "Flow run could not be submitted to infrastructure"
+                        f" {exception_traceback(exc)}."
+                    ),
+                )
```

Comment:
Hm this error is that the infrastructure configuration could not be retrieved. I'd keep this one as "FAILED" for now because it's a problem with the user's configuration.

Reply:
Thanks for the pointer. Restored this segment to the original state.